MvvmCross/Platforms/Ios/Presenters/Attributes/MvxDetailSplitViewPresentationAttribute.cs

```diff
@@ -1,14 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MS-PL license.
-// See the LICENSE file in the project root for more information.
-
-using MvvmCross.Presenters.Attributes;
-
-namespace MvvmCross.Platforms.Ios.Presenters.Attributes
-{
-    public class MvxDetailSplitViewPresentationAttribute : MvxBasePresentationAttribute
```

Comment:
Although I think this change makes sense, there's no information about this on the PR description. Please add it, it makes my life so much easier when writing migration docs. Also improves traceability